README.md

```diff
@@ -192,6 +192,7 @@ The following parameters are specific to each Codacy component.
 | `crow.replicaCount`                                 | Number of replicas                                                                                  | `1`                                          |
 | `crow.image.repository`                             | Image repository                                                                                    | from dependency                              |
 | `crow.image.tag`                                    | Image tag                                                                                           | from dependency                              |
+| `crow.config.crow.url`                              | Url pointing to crow                                                                                | `nil`                                        |
```

Comment:
Shouldn't this be global?

Reply:
Crow didn't have this url exposed through global, which is why this isn't global on this chart. Should we change crow to have this url configured from .Values.global.crow.url instead, and reproduce that here?

Reply:
Thoughts @rtfpessoa @paulopontesm ?

Reply:
yes, that is probably the best idea. Since it is used in website and crow.
I would say all urls should probably be in global.

Reply:
Updated in crow 4.9.1 👍